src/main/scala/io/keen/client/scala/AttemptCountingEventStore.scala

```diff
@@ -0,0 +1,13 @@
+package io.keen.client.scala
+
+import java.io.IOException
+
+trait AttemptCountingEventStore extends EventStore {
+  
+  @throws(classOf[IOException])
```

Comment:
Don't think these `@throws` annotations are necessary on the abstract trait methods?